src/wwwroot/app/views/settings/members/members.controller.js

```diff
@@ -66,7 +66,8 @@ export default function MembersController(
       vm.newUser = {
          roleId: role.id,
          login: '',
-         email: ''
+         email: '',
+         mailId: 1
```

Comment:
it is too hard code. Please, implement such more flexible. For example, you can set first id from templates list;